<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Причины ЧС</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#ffff7f
}
QPushButton{
background-color:#55aaff;
border-style:outset;
padding:2px;
border-width:5px;
border-radius:7px;
border-color:#55aaff;
}
</string>
  </property>
  <widget class="QTableView" name="tVFeature">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tVPossible">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>271</y>
     <width>661</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>132</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="BtnAddFeature">
      <property name="toolTip">
       <string>Добавить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/add-file.png</normaloff>../../../Downloads/add-file.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDelFeature">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Удалить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/delete.png</normaloff>../../../Downloads/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSaveFeature">
      <property name="toolTip">
       <string>Сохранить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/diskette.png</normaloff>../../../Downloads/diskette.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>132</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="BtnAddPossible">
      <property name="toolTip">
       <string>Добавить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/add-file.png</normaloff>../../../Downloads/add-file.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDelPossible">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Удалить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/delete.png</normaloff>../../../Downloads/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSavePossible">
      <property name="toolTip">
       <string>Сохранить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/diskette.png</normaloff>../../../Downloads/diskette.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
